### PR TITLE
feat: add Noto HK fonts to conf

### PR DESCRIPTION
**Pull request by Pi-Cla**
- Hong Kong and Macau are very close both physically and culturally so they should both prioritize Noto HK fonts
- Also changed the position of Noto KR to be higher for JP cause I think it should be higher than the Chinese variants of Noto for JP
- ~~Note that this is affected by https://bugzilla.opensuse.org/show_bug.cgi?id=1201571 as currently, Noto Serif CJK HK fonts are not installable on openSUSE. (but not blocked as that just means for now zh-hk and zh-mo users will default to Noto Serif CJK TC fonts)~~
-  The above has been fixed now as Noto Serif CJK HK fonts are now available

### CODE REVIEW

**Comment by ddemaio** [comment]
@m4u9 you want to add anything to this or approve it?

**Comment by m4u9** [comment]
As fellow HKer LGTM :). However I cannot really comment on the KR repositioning

**Comment by m4u9** [comment]
@ddemaio for the record I do not have merge permissions in this repo

**Comment by ddemaio** [comment]
@m4u9 me neither. I only see a few people who contribute here. @guoyunhe & @hellcp  along with antlarr above.

**Comment by antlarr** [comment]
@ddemaio I'm afraid I don't have write permissions either in this project. Probably @qiangzhao or @marguerite can help here.

**Comment by qiangzhao** [comment]
I disagree with updating font settings before we deliver the font, and even after that, there should have a user-level bug first.
Please don't say: "Hong Kong and Macau are very close both physically and culturally so they should both prioritize Noto HK fonts..."
Please refer it like:
1, install openSUSE Factory(leap) xxx;
2, the user does some operation;
3, ...
Result:
check the output
there has a font "X" type incorrect.
Resolution:
with your modification, the problem disappears with no side effects.
In other words, you have to give out a real user-level failure.

Thank you for considering my suggestions.
